orchestration/dags/data_gcp_dbt/macros/generate_schema_name.sql

```diff
@@ -5,21 +5,21 @@
 
         {{ default_schema }}
         
-    {%- elif custom_schema_name and  ((target.name == "prod" and target.profile_name != "sandbox") or target.name == "stg") -%}
+    {%- elif custom_schema_name and  ((target.name == "prod" and target.profile_name != "sandbox") or target.name == "stg" or target.name == "dev") -%}
```

Comment:
elif custom_schema_name and  (target.profile_name != "sandbox" or target.name != "local")

?